src/services/navigationBar.ts

```diff
@@ -138,7 +138,13 @@ namespace ts.NavigationBar {
         }
 
         function sortNodes(nodes: Node[]): Node[] {
-            return nodes.slice(0).sort((n1: Declaration, n2: Declaration) => {
+            const sortedCopy = nodes.slice(0);
+            doSortNodes(sortedCopy);
+            return sortedCopy;
+        }
+
+        function doSortNodes(nodes: Node[]): void {
+            nodes.sort((n1: Declaration, n2: Declaration) => {

```

Comment:
You're sorting by `Node`s but this comparison function assumes it has `Declaration`s?